libtcmu.c

```diff
@@ -438,9 +440,10 @@ static bool device_parse_cfg(struct tcmu_device *dev,
 	/* Check valid cfgstring */
 	oldptr = cfgstring;
 	ptr = strchr(oldptr, '/');
+	cfgstrlen = ptr-oldptr;
```

Comment:
Why not just use the `len` instead of adding a new `cfgstrlen` ?

Reply:
Done